src/DSi.h

```diff
@@ -129,6 +129,8 @@ class DSi final : public NDS
     void ARM7IOWrite16(u32 addr, u16 val) override;
     void ARM7IOWrite32(u32 addr, u32 val) override;
 
+    bool GetSCFGRasterBit() override;
```

Comment:
Could you mark this method as `const` so that it can be used with `const` references?

Reply:
it is const now